torch/_subclasses/fake_tensor.py

```diff
@@ -587,6 +587,9 @@ class FakeTensor(Tensor):
     item_memo = UnbackedMemoDescriptor()
     unique_memo = UnbackedMemoDescriptor()
 
+    _nested_int_memo: Optional[torch.SymInt]
+    _nested_int_memo_vc: Optional[int]
```

Comment:
Some adjustments are needed, but seems not too bad.  1) Made it more generic rather than unbacked specific, 2) I want to be able to operate generically on both FakeTensor and FunctionalTensor so had to add a "PassThroughDescriptor" there.